### ISSUE TITLE
how to resolve: requests.exceptions.HTTPError: 401 Client Error: UNAUTHORIZED for url: https://botometer.osome.iu.edu/api-test/4/check_account

### ISSUE THREAD

**Issue by abubakar-ucr** [IUNetSci/botometer-python#57]
I am on free account and getting this error while running the starter code from git repo:
requests.exceptions.HTTPError: 401 Client Error: UNAUTHORIZED for url: https://botometer.osome.iu.edu/api-test/4/check_account

**Comment by yangkcatiu** [comment]
Please resubscribe to one of the new plans on RapidAPI.

Also, there is a bug associated with the Basic plan on the RapidAPI end right now. Your case seems to be related. We are in communication with them to try to fix the bug. Although I'm not sure when we can get it done.

**Comment by abubakar-ucr** [comment]
unsubscribe and subscribe does not seem to help? Is there some other way that I can try out your API for free?

**Comment by filmenczer** [comment]
We're aware of a problem with the Basic plan. We're trying to work with
RapidAPI to fix it. Sorry for the inconvenience.

-Botometer Team

**Comment by yangkcatiu** [comment]
Could you try again?

**Comment by abubakar-ucr** [comment]
Thanks @yangkcatiu, it is working now. I will close the issue.

**Comment by javier-pg** [comment]
I was also experimenting the UNAUTHORIZED error, but today it works! Thanks for the correction

However, I have noticed that the quota of the basic plan is reached too early, without overpassing the number of 500 requests.  For example, with the total of 6 requests (seen on RapidAPI), the limit error is raised (on code and also in the notification of RapidAPI interface). I have also employed delay on requests, so it seems not to be a problem of Twitter rate limit...
